### ISSUE TITLE
Speed up rendering with dot symbolizer

### ISSUE THREAD

**Issue by gisuser1024** [mapnik/mapnik#3498]
I am using node js/ mapnik to render a png file and serve back to a client.   For about a million points I am seeing close to 10 second rendering times.  (Basically about 100K per second)    I am using the new dotsymbolizer.   width=1.5 and height=1.5 (nothing else).  The majority of points if any do not overlap.  I'm on an 8CPU, 64G Ram server.    Is this the expected time for about a million points?      Is there a way to speed things up.  Any internal settings, best practices, tricks, etc...  I don't need any real special styling.  Just a single colored dot on a map. 

Basically I'm pulling data from an external source into a CSV Memory data store and using mapnik to render. (a PNG tile)   

I'm debugging my code and can see that this is the actual Render time, and not query time.
THanks for your help!
